### PR TITLE
rclone: update page

### PR DESCRIPTION
**Pull request by soleofthesea**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/rclone.md [hunk 1]
```diff
@@ -3,27 +3,27 @@
...
+- Launch an interactive menu to setup rclone:
+
+`rclone config`
+
 - List contents of a directory on an rclone remote:
 
 `rclone lsf {{remote_name}}:{{path/to/directory}}`
 
-- Copy file or directory from local source to remote destination:
+- Copy file or directory from the local machine to remote destination:
```
Comment (kbdharun) [inline pages/common/rclone.md:14]:
```suggestion
- Copy file or directory from the local machine to the remote destination:
```

File: pages/common/rclone.md [hunk 2]
```diff
@@ -3,27 +3,27 @@
...
 
-`rclone copy {{path/to/source_file_or_directory}} {{remote_name}}:{{path/to/destination_directory}}`
+`rclone copy {{path/to/source_file_or_directory}} {{remote_name}}:{{path/to/directory}}`
 
-- Copy file or directory from remote source to local destination:
+- Copy files changed within the past 24 hours to a remote to the local machine, asking user to confirm each file:
```
Comment (kbdharun) [inline pages/common/rclone.md:18]:
```suggestion
- Copy files changed within the past 24 hours to a remote from the local machine, asking the user to confirm each file:
```
Is the copying happening from a remote to the local machine, or from a local machine to the remote?
Feel free to reword my suggestion for the same.

File: pages/common/rclone.md [hunk 3]
```diff
@@ -3,27 +3,27 @@
...
 
-`rclone copy {{remote_name}}:{{path/to/source_file_or_directory}} {{path/to/destination_directory}}`
+`rclone copy --interactive --max-age 24h {{remote_name}}:{{path/to/directory}} {{path/to/local_directory}} `
 
-- Sync local source to remote destination, changing the destination only:
+- Mirror file or directory (unlike copy, sync removes files from remote if it does not exist on local):
```
Comment (kbdharun) [inline pages/common/rclone.md:22]:
```suggestion
- Mirror file or directory (unlike copy, sync it removes files from remote if it does not exist locally):
```

File: pages/common/rclone.md [hunk 4]
```diff
@@ -3,27 +3,27 @@
...
 
 `rclone sync {{path/to/file_or_directory}} {{remote_name}}:{{path/to/directory}}`
 
-- Move file or directory from local source to remote destination:
-
-`rclone move {{path/to/file_or_directory}} {{remote_name}}:{{path/to/directory}}`
-
-- Delete remote file or directory (use `--dry-run` to test, remove it to actually delete):
+- Delete remote file or directory (`--dry-run` means test, remove it from the command to actually delete):
```
Comment (kbdharun) [inline pages/common/rclone.md:26]:
```suggestion
- Delete remote file or directory (Note: `--dry-run` means test, remove it from the command to actually delete):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

----

Just a minor suggestion, in future I would suggest using a shorter and more generic commit title (you could describe all your changes in the commit message/PR's description). I have updated the same here, feel free to checkout https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message.

File: pages/common/rclone.md (hunk 3 above)
Comment (soleofthesea) [reply pages/common/rclone.md:22]: I believe the word "it" would make it ungrammatical in this case (also for consistency "Note:" should also be added)

File: pages/common/rclone.md [hunk 5]
```diff
@@ -3,27 +3,27 @@
...
+- Launch an interactive menu to setup rclone:
+
+`rclone config`
+
 - List contents of a directory on an rclone remote:
 
 `rclone lsf {{remote_name}}:{{path/to/directory}}`
 
-- Copy file or directory from local source to remote destination:
+- Copy file or directory from the local machine to the remote destination:
```
Comment (vitorhcl) [inline pages/common/rclone.md:14]:
```suggestion
- Copy a file or directory from the local machine to the remote destination:
```

File: pages/common/rclone.md [hunk 6]
```diff
@@ -3,27 +3,27 @@
...
+- Mirror file or directory (Note: Unlike copy, sync removes files from the remote if it does not exist locally):
 
 `rclone sync {{path/to/file_or_directory}} {{remote_name}}:{{path/to/directory}}`
 
-- Move file or directory from local source to remote destination:
-
-`rclone move {{path/to/file_or_directory}} {{remote_name}}:{{path/to/directory}}`
-
-- Delete remote file or directory (use `--dry-run` to test, remove it to actually delete):
+- Delete remote file or directory (Note: `--dry-run` means test, remove it from the command to actually delete):
```
Comment (vitorhcl) [inline pages/common/rclone.md:26]:
```suggestion
- Delete a remote file or directory (Note: `--dry-run` means test, remove it from the command to actually delete):
```

File: pages/common/rclone.md [hunk 7]
```diff
@@ -3,27 +3,27 @@
...
+`rclone copy {{path/to/source_file_or_directory}} {{remote_name}}:{{path/to/directory}}`
 
-- Copy file or directory from remote source to local destination:
+- Copy files changed within the past 24 hours to a remote from the local machine, asking the user to confirm each file:
 
-`rclone copy {{remote_name}}:{{path/to/source_file_or_directory}} {{path/to/destination_directory}}`
+`rclone copy --interactive --max-age 24h {{remote_name}}:{{path/to/directory}} {{path/to/local_directory}} `
 
-- Sync local source to remote destination, changing the destination only:
+- Mirror file or directory (Note: Unlike copy, sync removes files from the remote if it does not exist locally):
```
Comment (vitorhcl) [inline pages/common/rclone.md:22]:
```suggestion
- Mirror a specific file or directory (Note: Unlike copy, sync removes files from the remote if it does not exist locally):
```

**Review by vitorhcl — CHANGES_REQUESTED** [review]
Thank you for your contribution. I have some suggestions

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution.

LGTM, after the above suggestions.

**Review by vitorhcl — APPROVED** [review]

**Pull request merged**